code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -247,75 +258,69 @@
 	var/deployment_cooldown
 	var/obj/machinery/marine_turret/premade/dropship/deployed_turret
 
-	initialize()
-		if(!deployed_turret)
-			deployed_turret = new(src)
-			deployed_turret.deployment_system = src
-		..()
-
-	New()
-		if(!deployed_turret)
-			deployed_turret = new(src)
-			deployed_turret.deployment_system = src
-		..()
+/obj/structure/dropship_equipment/sentry_holder/New()
+	if(!deployed_turret)
+		deployed_turret = new(src)
+		deployed_turret.deployment_system = src
+	return ..()
 
-	examine(mob/user)
-		..()
-		if(!deployed_turret)
-			to_chat(user, "Its turret is missing.")
+/obj/structure/dropship_equipment/sentry_holder/examine(mob/user)
+	..()
+	if(!deployed_turret)
+		to_chat(user, "Its turret is missing.")
 
-	on_launch()
-		undeploy_sentry()
+/obj/structure/dropship_equipment/sentry_holder/on_launch()
+	undeploy_sentry()
 
-	equipment_interact(mob/user)
-		if(deployed_turret)
-			if(deployment_cooldown > world.time)
-				to_chat(user, "<span class='warning'>[src] is busy.</span>")
-				return //prevents spamming deployment/undeployment
-			if(deployed_turret.loc == src) //not deployed
-				if(z == LOW_ORBIT_Z_LEVEL)
-					to_chat(user, "<span class='warning'>[src] can't deploy mid-flight.</span>")
-				else
-					to_chat(user, "<span class='notice'>You deploy [src].</span>")
-					deploy_sentry()
+/obj/structure/dropship_equipment/sentry_holder/equipment_interact(mob/user)
+	if(deployed_turret)
```

Comment:
if(!deployed_turret)
 to_chat(user, "<span class='warning'>[src] is unresponsive.</span>")
 return